src/components/accordion/accordion.tsx

```diff
@@ -70,7 +70,8 @@ export class Accordion {
       <div
         class={{
           "accordion--transparent": this.appearance === "transparent",
-          "accordion--minimal": this.appearance === "minimal"
+          "accordion--minimal": this.appearance === "minimal",
+          "accordion--default": this.appearance === "default"
```

Comment:
Nitpick: since this CSS class is the default, maybe it should just be `accordion`?